From a8ec122816fdbcc1619b61990e8b306e1f071f50 Mon Sep 17 00:00:00 2001
From: Bart Ribbers <bribbers@disroot.org>
Date: Sun, 6 Sep 2020 14:39:27 +0200
Subject: [PATCH] Fix mycroft-gui-core loader and stop

The conditional statement in the stop script was never actually closed

Also fix some shellcheck issues while we're at it
---
 data/mycroft-gui-core-loader.in | 2 +-
 data/mycroft-gui-core-stop.in   | 3 ++-
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/data/mycroft-gui-core-loader.in b/data/mycroft-gui-core-loader.in
index c65de79..39bb605 100644
--- a/data/mycroft-gui-core-loader.in
+++ b/data/mycroft-gui-core-loader.in
@@ -5,7 +5,7 @@ if ! test -z "$MYCROFT_START_CMD"; then
 	$MYCROFT_START_CMD
 else
 	# Use default git install location
-	cd ${MYCROFT_CORE_DIR}
+	cd "${MYCROFT_CORE_DIR}" || exit
 	./start-mycroft.sh all
 fi
 
diff --git a/data/mycroft-gui-core-stop.in b/data/mycroft-gui-core-stop.in
index 65a5cef..9e078f9 100644
--- a/data/mycroft-gui-core-stop.in
+++ b/data/mycroft-gui-core-stop.in
@@ -4,5 +4,6 @@ if ! test -z "$MYCROFT_STOP_CMD"; then
 	$MYCROFT_STOP_CMD
 else
 	# Use default git install
-	cd ${MYCROFT_CORE_DIR}
+	cd "${MYCROFT_CORE_DIR}"|| exit
 	./stop-mycroft.sh
+fi
